src/TodoListItem.jsx

```diff
@@ -1,6 +1,6 @@
-const TodoListItem = (props) => {
+const TodoListItem = ({todo}) => {
     return (
-        <li>{props.todo}</li>
+        <li>{todo}</li>
```

Comment:
When running your application, make sure you watch your console. React will be giving you a warning about these list items not having an id. When mapping over a value that creates several sibling elements, React needs to be able to track all of the components that it renders and uses `key={todo.id}` to do so.

If we do not do this, React is forced to update all of the sibling components every time any one of them are updated. This is very bad for performance.

Reply:
I believe that I fixed this issue by having the TodoListItem function return the following line of code instead:
`<li key={todo.id}>{todo}</li>`

Reply:
You can find these changes in at [my lesson_1_5_b branch ](https://github.com/NatalyBMota/react-todo/tree/lesson_1_5_b).